src/commands/admin/setassign.ts

```diff
@@ -1,36 +1,45 @@
-import { Message, Role } from 'discord.js';
+import { ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 import { AssignableRole } from '@lib/types/AssignableRole';
-import { roleParser } from '@lib/arguments';
-import { adminPerms } from '@lib/permissions';
+import { ADMIN_PERMS } from '@lib/permissions';
 import { DB } from '@root/config';
 import { Command } from '@lib/types/Command';
+import { modifyRoleDD } from '@root/src/lib/utils';
 
 export default class extends Command {
 
 	description = `Adds a role to the assignable collection of the database, or removes it if it's there already`;
 	usage = '<role>';
-	aliases = ['addassign'];
 	runInDM = false;
+	tempPermissions: ApplicationCommandPermissionData[] = [ADMIN_PERMS];
+	options: ApplicationCommandOptionData[] = [{
+		name: 'role',
+		description: 'The role to add to assignables.',
```

Comment:
The description of this command sounds kind of vague IMO. I know this is meant to convey that it'll add the role to the list of self-assignable roles, maybe change it to reflect that intention?